tests/kernel/ib_tests.pm

```diff
@@ -114,7 +114,6 @@ sub run {
     my $role = get_required_var('IBTEST_ROLE');
 
     select_console 'root-ssh';
-    zypper_call('ar -f -G ' . get_required_var('GA_REPO'));
```

Comment:
In this case, the zypper_call('--gpg-auto-import-keys ref'); could(should) be moved into the if-block, because there is no need to refresh for the slaves. Anyway, it would not do any harm. We can clean this later as well.

Reply:
I leave this for you, you could rebase and add it to PR 5313